bindings/binding_core_node/src/minify.rs

```diff
@@ -86,7 +109,11 @@ fn minify(code: Buffer, opts: Buffer, signal: Option<AbortSignal>) -> AsyncTask<
 }
 
 #[napi]
-pub fn minify_sync(code: Buffer, opts: Buffer) -> napi::Result<TransformOutput> {
+pub fn minify_sync(
+    code: Buffer,
+    opts: Buffer,
+    error_format: Option<ErrorFormatConfig>,
```

Comment:
Also, why is this an additional parameter, instead of additional field?

Reply:
ı think reasoning behind using an additional parameter instead of a field was to maintain the separation of concerns. By keeping ErrorFormatConfig as a separate argument, it avoids bloating the existing options structure and allows for easier future extensions or modifications